core/primitives/src/runtime/config_store.rs

```diff
@@ -198,4 +203,20 @@ mod tests {
             &serde_json::from_slice::<RuntimeConfig>(CONFIGS[2].1).unwrap()
         );
     }
+
+    #[test]
+    #[cfg(feature = "protocol_feature_lower_ecrecover_base_cost")]
+    fn test_lower_ecrecover_base_cost() {
+        let store = RuntimeConfigStore::new(None);
+        let base_cfg = store.get_config(LowerStorageCost.protocol_version());
+        let new_cfg = store.get_config(LowerEcrecoverBaseCost.protocol_version());
+        assert!(
+            base_cfg.transaction_costs.data_receipt_creation_config.base_cost.send_sir
+                > new_cfg.transaction_costs.data_receipt_creation_config.base_cost.send_sir
+        );
+        assert!(
+            base_cfg.transaction_costs.data_receipt_creation_config.cost_per_byte.send_sir
+                > new_cfg.transaction_costs.data_receipt_creation_config.cost_per_byte.send_sir
+        );
+    }
```

Comment:
This test feel a bit like busy work, to be honest, I am not sure it adds much value. 

Thinking about this more.... Why didn't this change failed any of the existing tests? It seems that we should have a test that ecrecover behaves as expected at the latest version of the protocol, to make sure we don't change it by accident.

Reply:
I also missed changing the config value I wanted to test when I was copying the test from #4795 🤦‍♂️ Obviously this test was meant to test the ecrecover base cost value. Fixed in https://github.com/near/nearcore/pull/4865/commits/d4668b0361991b6f134a8a59bf0c324a3d50db1c

>  to make sure we don't change it by accident.

I think it would fail the [test checking the config hashes](https://github.com/near/nearcore/blob/master/core/primitives/src/runtime/config_store.rs#L111) if I were to change an existing config as opposed to adding a new one.

Reply:
>I think it would fail the test checking the config hashes if I were to change an existing config as opposed to adding a new one.

Yeah, that's what I mean -- we have the test for config, but we don't have the test that verifies that the final cost is what we'd expect. 

https://github.com/near/nearcore/blob/43dc0d15190d25f8ec01dbd330bb7eb956826e6f/runtime/near-vm-logic/tests/test_miscs.rs#L589-L624 sort of does that, but it looks at what costs were used, not at the final gas cost. I think that's maybe though.